docker/bigtop-puppet/build.sh

```diff
@@ -61,7 +61,8 @@ case "${OS}-${VERSION}" in
     ;;
   openeuler-22.03*)
     OPENEULER_OS="${OS}/${OS}"
-    sed -i -e "s|${OS}:${VERSION}|$OPENEULER_OS:${VERSION}|" ./Dockerfile
+    # BIGTOP-4082 specify the openEuler OS version is : 22.03-lts-sp1
+    sed -i -e "s|${OS}:${VERSION}|$OPENEULER_OS:${VERSION}-lts-sp1|" ./Dockerfile
```

Comment:
@MacChen01 How about setting the VERSION by `VERSION=$(echo "$1" | cut -d '-' -f 3-)` in line 32? We can avoid hardcoding the version into the script by that. I'm expecting we can specify the version like

```
$ gradlew bigtop-puppet -POS=openeuler-22.03-lts-sp1 -Pprefix=3.3.0`
```

Reply:
@iwasakims Thanks for your review.
The version of openEuler 22.03 has many different subversions, we chose the 22.03-lts-sp1 version for openeuler-22.03.
So we hardcoding the version into the script by that.

Reply:
I don't like to update the code every time we change the version used (e.g. 22.03-lts-sp2 in the future).

Reply:
In addition, we should publish the image as openeuler-22.03-lts-sp1 rather than openeuler-22.03 if the base image is different.

Reply:
ok, i will modify the version follow your advice.